### DISCUSSION TITLE
Auto table of contents for books like bookdown

### DISCUSSION THREAD

**Discussion by sbalci** [quarto-dev/quarto-cli#738] [Feature Requests]
Hi,

With bookdown one can put Rmd files in project folder and the table of contents are genarated with all Rmd files, rendering all of them with alphabetic order.
Would it be possible to mimmic that action with Quarto Books?

Best wishes

Serdar

**Comment by jjallaire** [comment]
Yes, this is something we haven't implemented yet but definitely plan to!

**Comment by mcanouil** [comment]
If I am not mistaken, this is implemented in current version.

**Comment by sbalci** [comment]
would you please guide me how to change the yml file to achieve this?

**Reply by mcanouil** [reply]
I suggest to use the latest version of Quarto to create a book (https://quarto.org/docs/books/).
As you can see in the documentation the left menu will include the files in the menu, but the actual TOC (by default on the right) is independent by design.
